tests/page-objects/common/common.wdio.page.js

```diff
@@ -12,6 +12,14 @@ const modal = require('./modal.wdio.page');
 const loaders = () => $$('.container-fluid .loader');
 const syncSuccess = () => $(`${hamburgerMenuItemSelector}.sync-status .success`);
 const reloadModalCancel = () => $('#update-available .btn.cancel:not(.disabled)');
+
+//languages
+const languagePreferenceHeading = () => $('#language-preference-heading');
+const selectedPreferenceHeading = () => $('#language-preference-heading > h4:nth-child(1) > span:nth-child(3)');
+const messagesLanguage = () => $('.locale a.selected span.rectangle');
```

Comment:
I don't think the `rectangle` styling class provides much value in this selector. Can it be omitted and just use `'.locale a.selected span'? Same for the selector below.